src/caller.jl

```diff
@@ -0,0 +1,189 @@
+
+function recursive_translate(x, R)
+    if length(x) > 0 && x[1] isa Bool
+        return convert_return_list(x, R)
+    else
+        return [ recursive_translate(i, R) for i in x]
+    end
+end
+
+# function LIST_CMD_CASTER(x)
+#     x_new = LIST_CMD_TRAVERSAL(x)
+# end
+
+casting_functions_pre = Dict(:NUMBER_CMD     => (libSingular.NUMBER_CMD_CASTER,     true, ()),
```

Comment:
Looking at this, I guess I don't like the VS code auto-formatter that much :-/

Reply:
Yeah, it produces terribly ugly code. On the other hand, it is a standard. I am not sure which I like more, tbh.

Reply:
You must have an option set incorrectly. It can't be a standard to use single line comments everywhere. I use VSCode and I use multiline comments.

Reply:
That has nothing to do with the comments, but with the dict input formatting. I put the single line comments, because that is my setting, and it is not incorrect at all.

However, the code formatting does not allow for settings. So here we are.